### PR TITLE
add -r parameter for create and init command

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java [hunk 1]
```diff
@@ -55,12 +55,18 @@
 
 	public static final String TEMPLATES_VERSION = "1+";
 
+	private static List<String> templateNames;
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: We don't need to introduce these new static variables.  All the `-r` should do is determine if `--refresh-dependencies` is called or not.  That is all.

File: com.liferay.blade.cli/src/com/liferay/blade/cli/InitCommand.java [hunk 2]
```diff
@@ -51,6 +51,8 @@ public InitCommand(blade blade, InitOptions options) throws Exception {
 	}
 
 	public void execute() throws IOException {
+		GradleTooling.setRefresh(_options.refresh());
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/InitCommand.java]: We don't need a "setRefresh".  just add a boolean in the findLatestAvailableArtifact() method.

File: com.liferay.blade.cli/src/com/liferay/blade/cli/gradle/GradleTooling.java [hunk 3]
```diff
@@ -38,6 +39,8 @@
  */
 public class GradleTooling {
 
+	private static boolean refresh = false;
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/gradle/GradleTooling.java]: Add refresh as a argument to the method not as a field in this class.

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java (hunk 1 above)
Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: ok

File: com.liferay.blade.cli/src/com/liferay/blade/cli/InitCommand.java (hunk 2 above)
Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/InitCommand.java]: ok

**Pull request closed without merging**